coding/browser-support.md

```diff
@@ -4,20 +4,20 @@ layout: base
 
 # Browser support
 
-Our baseline browser support for our websites. There are two levels of support. 
+Our baseline browser support for our websites. There are two levels of support.
 
 ## Full support
 
 Any browser with full support needs to be pixel perfect and all functionality should work.
 
 | Browser | Version |
 | ------- | ------- |
-| Chrome | 55 or greater |
-| Firefox | 52 or greater |
-| Safari | 11 or greater |
+| Chrome | Should work for the last 3 [stable versions](https://www.chromestatus.com/features/schedule) |
```

Comment:
Apologies, @anthonydillon, I realised this should probably have the word "major" in it.

```suggestion
| Chrome | Should work for the last 3 [stable major versions](https://www.chromestatus.com/features/schedule) |
```